From eb59e787e13a66196412d2ed2b3771092f3c799e Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Sat, 22 Sep 2018 11:17:41 +0530
Subject: [PATCH 0208/1051] soc: octeontx2: Remove CGX rxtx disable during
 nix_lf_teardown

Since packet gating is done at NPC level, we dont need to
turn off rxtx at CGX level during NIX LF teardown.

Signed-off-by: Linu Cherian <lcherian@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2/rvu_nix.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/soc/marvell/octeontx2/rvu_nix.c b/drivers/soc/marvell/octeontx2/rvu_nix.c
index f07606ba8476..1e48b893cfae 100644
--- a/drivers/soc/marvell/octeontx2/rvu_nix.c
+++ b/drivers/soc/marvell/octeontx2/rvu_nix.c
@@ -2598,9 +2598,6 @@ void rvu_nix_lf_teardown(struct rvu *rvu, u16 pcifunc, int blkaddr, int nixlf)
 
 	ctx_req.hdr.pcifunc = pcifunc;
 	req.hdr.pcifunc = pcifunc;
-	/* Stop packet Rx/Tx */
-	req.hdr.id = MBOX_MSG_CGX_STOP_RXTX;
-	rvu_cgx_config_rxtx(rvu, pcifunc, false);
 
 	/* Cleanup NPC MCAM entries, free Tx scheduler queues being used */
 	nix_interface_deinit(rvu, pcifunc, nixlf);
-- 
2.17.1

